src/components/ListSection.tsx

```diff
@@ -1,3 +1,52 @@
+import { useEffect, useState } from 'react';
+import { instance } from '../libs/api';
+
+// records 데이터를 받아오기 위한 Records interface 생성
+interface Records {
+  id: string;
+  fields: Fields;
+}
```

Comment:
```suggestion
interface Records {
  id: string;
  fields: {
    name: string;
    }
}
```

이런 식으로 중첩해서 작성할 수 있어요!

근데 대부분의 상황에서는 현재 작성해주신 것처럼 계층에 따라 인터페이스를 분리하는 것을 권장드리고 싶긴해요!